src/command/contract/publish.rs

```diff
@@ -0,0 +1,125 @@
+use clap::{ArgGroup, Parser};
+use serde::Serialize;
+
+use crate::options::{GraphRefOpt, ProfileOpt};
+use crate::utils::client::StudioClientConfig;
+use crate::{RoverOutput, RoverResult};
+
+use rover_client::operations::contract::publish::{self, ContractPublishInput};
+use rover_client::RoverClientError;
+use rover_std::Style;
+
+#[derive(Debug, Serialize, Parser)]
+#[clap(
+    group = ArgGroup::new("include_tags_group")
+        .args(&["include_tag", "no_include_tags"]).required(true).multiple(false),
+    group = ArgGroup::new("exclude_tags_group")
+        .args(&["exclude_tag", "no_exclude_tags"]).required(true).multiple(false),
+    group = ArgGroup::new("hide_unreachable_types_group")
+        .args(&["hide_unreachable_types", "no_hide_unreachable_types"]).required(true).multiple(false)
```

Comment:
i think these `multiple(false)` calls here are redundant because the default is `false` ([src](https://docs.rs/clap/latest/clap/struct.ArgGroup.html#method.multiple))

```suggestion
    group = ArgGroup::new("include_tags_group")
        .args(&["include_tag", "no_include_tags"]).required(true),
    group = ArgGroup::new("exclude_tags_group")
        .args(&["exclude_tag", "no_exclude_tags"]).required(true),
    group = ArgGroup::new("hide_unreachable_types_group")
        .args(&["hide_unreachable_types", "no_hide_unreachable_types"]).required(true)
```